point_cloud_client/src/bin/test.rs

```diff
@@ -54,8 +53,8 @@ fn main() {
     let num_points = args.num_points;
     let point_cloud_client = PointCloudClient::new(&args.locations, OctreeFactory::new())
         .expect("Couldn't create octree client.");
-    let point_location = PointLocation {
-        culling: PointCulling::Aabb(Aabb3::new(args.min, args.max)),
```

Comment:
Any specific reason to remove the min/max restriction?
Then the whole point of having min/max arguments goes away...

Reply:
done

Reply:
i think it also was a copy-paste error